aiogram/dispatcher/filters/builtin.py

```diff
@@ -12,6 +12,19 @@
 from aiogram.types import CallbackQuery, Message, InlineQuery, Poll, ChatType
 
 
+IDFilterArgumentType = typing.Union[typing.Iterable[typing.Union[int, str]], str, int]
```

Comment:
what about:
```
_UserID = typing.Union[str, int]
IDFilterArgumentType = typing.Union[typing.Iterable[_UserID], _UserID]
```
?

Reply:
I doubt if this good name. `IDFilterArgumentType` implies that this type is used only in IDFilter, but that's not the case here, is it? But LGTM
```python3
_ID = TypeVar("_ID", int, str)
IDArgumentType = Union[Sequence[_ID], _ID]
```

Reply:
I don't think name matters here, type is internal. However I agree sounds nicer if shorter.

Reply:
Changed type name to `ChatIDArgumentType`